Mammoth/Include/TSEAccessibilitySettings.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include "DirectXUtil.h"
+#include "Kernel.h"
+
+using namespace Kernel;
+
+class CAccessibilitySettings {
+public:
+	enum IFFType {
```

Comment:
Use enum class here, which is the new C++11 way of doing enums.

Reply:
upcoming commit (working on it right now) implements this

Reply:
done